sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/util/DateTimeUtilsSuite.scala

```diff
@@ -542,9 +542,8 @@ class DateTimeUtilsSuite extends SparkFunSuite with Matchers with SQLHelper {
     checkError(
       exception = intercept[SparkIllegalArgumentException](
         dateAddInterval(input, new CalendarInterval(36, 47, 1))),
-      condition = "_LEGACY_ERROR_TEMP_2000",
+      condition = "UNSUPPORTED_DATETIME_UNIT_ADDITION",
```

Comment:
How can we reach this code? I sthis non-SQL API only? SHoudl this be an internal error?

Reply:
hm... yeah I think it's an internal error. Let me update it